### PR TITLE
Clarify source of nextcloud entity information

### PR DESCRIPTION
**Pull request by meichthys**
I've had a few requests asking what sensors are created by this integration.

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nextcloud.markdown [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Nextcloud
-description: Instructions on how to integrate Nextcloud monitor api data into Home Assistant.
+description: Instructions on how to integrate Nextcloud monitor api data into Home Assistant. Sensors and BinarySensors will be created for most of the datapoints that the built-in Nextcloud [serverinfo app](https://github.com/nextcloud/serverinfo) provides. To see which datapoints your nextcloud instance exposes, browse to this url: `https://<your_nextcloud_url>/ocs/v2.php/apps/serverinfo/api/v1/info?format=json`
```
Comment (frenck) [inline source/_integrations/nextcloud.markdown]: I don't think this belongs to the meta description part here.

Please adjust the documentation below instead.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/nextcloud.markdown (hunk 1 above)
Comment (meichthys) [reply source/_integrations/nextcloud.markdown]: 🤦‍♂️Oh, right. Should be all set now.

File: source/_integrations/nextcloud.markdown [hunk 2]
```diff
@@ -15,7 +15,7 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The `nextcloud` integration pulls summary [Nextcloud](https://nextcloud.com/) information into Home Assistant.
+The `nextcloud` integration pulls summary [Nextcloud](https://nextcloud.com/) information into Home Assistant. Sensors and BinarySensors will be created for most of the datapoints that the built-in Nextcloud [serverinfo app](https://github.com/nextcloud/serverinfo) provides. To see which datapoints your nextcloud instance exposes, browse to this url: `https://<your_nextcloud_url>/ocs/v2.php/apps/serverinfo/api/v1/info?format=json`.
```
Comment (frenck) [inline source/_integrations/nextcloud.markdown]: Small tweak suggestion for readability

```suggestion
The `nextcloud` integration pulls summary [Nextcloud](https://nextcloud.com/) information into Home Assistant.

The integration provides sensors and binary sensors for most of the data points that the built-in Nextcloud [serverinfo app](https://github.com/nextcloud/serverinfo) provides.

To see which datapoints your nextcloud instance exposes, browse to this url: `https://<your_nextcloud_url>/ocs/v2.php/apps/serverinfo/api/v1/info?format=json`.
```

Comment (meichthys) [reply source/_integrations/nextcloud.markdown]: Looks good to me 👍

**Review by frenck — APPROVED** [review]
Thanks, @meichthys 👍

**Pull request merged**